fenix/app/src/main/java/org/mozilla/fenix/components/metrics/MetricsStorage.kt

```diff
@@ -217,6 +265,9 @@ internal class DefaultMetricsStorage(
         // The usage threshold we are interested in is currently 340 seconds.
         private const val usageThresholdMillis = 1000 * 340
 
+        // The usage threshold for "activated" growth users.
+        private const val daysActivatedThreshold = 3
```

Comment:
Since these names are already incorrectly cased (my bad), let's just leave these ones that you're adding as they are for now

Reply:
I'll leave the ones in `MetricsStorage` for now, might change them in the follow up ticket